CMakeLists.txt

```diff
@@ -2,17 +2,16 @@ cmake_minimum_required(VERSION 3.5)
 project(xacro)
 
 find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
 
 install(FILES scripts/completion.bash DESTINATION share/${PROJECT_NAME}/environment)
 
-## The following manually installs the python package and the entry-point script
-## Using ament_cmake, we cannot actually use setup.py to install these...
+# install xacro python package and it's console_script entry-point script
+ament_python_install_package(${PROJECT_NAME} SCRIPTS_DESTINATION lib/${PROJECT_NAME})
 
-# install xacro python package
-ament_python_install_package(xacro)
-# install main script
-install(PROGRAMS scripts/xacro DESTINATION bin)
-install(PROGRAMS scripts/xacro DESTINATION lib/${PROJECT_NAME})
+# install entry-point script(s) in bin as well
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/ament_cmake_python/${PROJECT_NAME}/scripts/xacro
+        DESTINATION bin)
```

Comment:
Using `PROGRAMS` instead breaks the build in windows:
```
CMake Error at cmake_install.cmake:68 (file):
  file INSTALL cannot find
  "C:/Users/melvi/git-repos/ros2_upstream_ws/build/xacro/ament_cmake_python/xacro/scripts/xacro":
  File exists.
  ```
  In windows, the generated file is `xacro.exe`, not `xacro`

Reply:
Argh. Hopefully the next commit fixes this.